frontend/src/pages/Dashboard/utils/schema.json

```diff
@@ -0,0 +1,245 @@
+{
```

Comment:
What is this file used for

Reply:
It's used to construct the register form